setup.py

```diff
@@ -31,16 +31,26 @@
     pip install git+https://github.com/in-toto/in-toto@develop
     ```
 """
+import os
+import re
+
 from setuptools import setup, find_packages
 
-version = "0.4.0"
+
+base_dir = os.path.dirname(os.path.abspath(__file__))
+
+def get_version(filename="in_toto/__init__.py"):
+  with open(os.path.join(base_dir, filename), encoding="utf-8") as initfile:
```

Comment:
Can you please use `io.open` instead of `open`. It backports the `encoding` parameter to Python 2. This will fix the [failing appveyor test](https://ci.appveyor.com/project/in-toto/in-toto/builds/27814466).

Reply:
I've used a version of this function for several (personal) projects over the years, but it seems that I explicitly decided against using `io.open` (and `codecs.open`?), but I don't remember why. Perhaps it was Python 3 issues? In any case, I'm happy to swap it as you get to maintain it.